ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerDemoController.swift

```diff
@@ -32,6 +32,7 @@ class DrawerDemoController: DemoController {
 
         addTitle(text: "Bottom Drawer")
         container.addArrangedSubview(createButton(title: "Show resizable", action: #selector(showBottomDrawerButtonTapped)))
+        container.addArrangedSubview(createButton(title: "Show resizable with expand state and passthrough behavior", action: #selector(showBottomDrawerWithExpandStateAndPassthroughBackgroundButtonTapped)))
```

Comment:
How about just "Show with underlying interactable content view" as the description?